modules/apps/portal-cache/portal-cache-test/src/testIntegration/java/com/liferay/portal/cache/internal/test/PortalCacheExtenderTest.java

```diff
@@ -63,24 +64,60 @@ public class PortalCacheExtenderTest {
 	public static final AggregateTestRule aggregateTestRule =
 		new LiferayIntegrationTestRule();
 
-	@Before
-	public void setUp() throws Exception {
-		_multiVmXML = _generateXMLContent(_TEST_CACHE_MULTI, 1001, 51);
-		_singleVmXML = _generateXMLContent(_TEST_CACHE_SINGLE, 1001, 51);
-
-		_bundle = _installBundle(
-			_BUNDLE_SYMBOLIC_NAME, _multiVmXML, _singleVmXML);
-	}
-
 	@After
 	public void tearDown() throws Exception {
 		if (_bundle.getState() != Bundle.UNINSTALLED) {
 			_bundle.uninstall();
 		}
 	}
 
+	@Test
+	public void testRecreateMultiVmConfig() throws Exception {
+		String multiCacheConfigName = "module-multi-vm.xml";
+
+		_multiVmXML = _getResourceAsString(
```

Comment:
Technically we can still write the content within a function and it would be the cleanest, and add variable arguments, it would be a lot more work, esp for only 1 more usage, its debatable so I won't object to using a resource file, but I know the devs/bchan did not like using resource files in the past so they may object to it fyi